dist/stat_data.py

```diff
@@ -813,16 +813,15 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_next_hs_tombstone', 'cursor next calls that skip due to a globally visible history store tombstone'),
     CursorStat('cursor_next_skip_ge_100', 'cursor next calls that skip greater than or equal to 100 entries'),
     CursorStat('cursor_next_skip_lt_100', 'cursor next calls that skip less than 100 entries'),
-    CursorStat('cursor_next_skip_page_count', 'Total number of pages skipped without reading by cursor next calls'),
     CursorStat('cursor_next_skip_total', 'Total number of entries skipped by cursor next calls'),
     CursorStat('cursor_open_count', 'open cursor count', 'no_clear,no_scale'),
     CursorStat('cursor_prev_hs_tombstone', 'cursor prev calls that skip due to a globally visible history store tombstone'),
     CursorStat('cursor_prev_skip_ge_100', 'cursor prev calls that skip greater than or equal to 100 entries'),
     CursorStat('cursor_prev_skip_lt_100', 'cursor prev calls that skip less than 100 entries'),
-    CursorStat('cursor_prev_skip_page_count', 'Total number of pages skipped without reading by cursor prev calls'),
     CursorStat('cursor_prev_skip_total', 'Total number of entries skipped by cursor prev calls'),
     CursorStat('cursor_search_near_prefix_fast_paths', 'Total number of times a search near has exited due to prefix config'),
     CursorStat('cursor_skip_hs_cur_position', 'Total number of entries skipped to position the history store cursor'),
+    CursorStat('cursor_skip_page_count', 'Total number of pages skipped without reading by cursor next or prev calls'),
```

Comment:
I am not 100% sure how useful this statistic will be in its current form. Unless we know if we are skipping internal pages or leaf pages, we can't compare values and say skipping X is better than skipping Y. But it is still helpful in giving us an indication of when a search is skipping pages. I am not keen on adding 2 statistics here, one for leaf pages and the other for internal pages, considering there might be a performance impact with adding them here.

Reply:
Agreed, removed the stat.